<ui version="4.0" >
 <class>OpenViewDialog</class>
 <widget class="QDialog" name="OpenViewDialog" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Open View</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="pmchart.qrc" >:/images/view.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <layout class="QGridLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item row="1" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4" >
      <widget class="QPushButton" name="openPushButton" >
       <property name="text" >
        <string>Open</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QComboBox" name="sourceComboBox" />
     </item>
     <item row="2" column="4" >
      <widget class="QPushButton" name="cancelPushButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>29</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="fileNameLabel" >
       <property name="text" >
        <string>File:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="sourceLabel" >
       <property name="text" >
        <string>Source:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" >
      <widget class="QPushButton" name="sourcePushButton" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="pmchart.qrc" >:/images/computer.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" >
      <widget class="QLineEdit" name="fileNameLineEdit" />
     </item>
     <item row="1" column="3" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>32</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="pathLabel" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pathComboBox" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>8</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="parentToolButton" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Parent</string>
       </property>
       <property name="statusTip" >
        <string/>
       </property>
       <property name="whatsThis" >
        <string>Open the parent directory</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="pmchart.qrc" >:/images/go-previous.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>8</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="userToolButton" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>User Views</string>
       </property>
       <property name="whatsThis" >
        <string>Display your personal saved Views.  A view is one or more charts.</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="pmchart.qrc" >:/images/toolusers.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="systemToolButton" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>System Views</string>
       </property>
       <property name="whatsThis" >
        <string>Display preconfigured system Views.  A view is one or more charts.</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="pmchart.qrc" >:/images/view.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QListView" name="dirListView" >
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="flow" >
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="isWrapping" stdset="0" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
     <property name="gridSize" >
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="uniformItemSizes" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pmchart.qrc" />
 </resources>
</ui>
